python/lsst/ap/association/assoc_afw_utils.py

```diff
@@ -0,0 +1,279 @@
+#
```

Comment:
Why not call the file `afwUtils.py`? It's already in `ap.association`, so `ap.association.assoc_afw_utils` seems redundant. Also, files are camelCase:

https://developer.lsst.io/python/style.html#a-python-source-file-name-should-be-camelcase-with-leading-lowercase-and-ending-in-py